### DISCUSSION TITLE
Get an error when using DirectML

### DISCUSSION THREAD

**Discussion by mariosconsta** [microsoft/tensorflow-directml-plugin#227] [Q&A]
I have an AMD GPU and I came upon this tool so I thought I could try it out! I did pip install tensorflow-directml, restarted my kernel and now I get this error when I run this code:

```
a = tf.constant([1.0, 2.0, 3.0]) 

b = tf.constant([4.0, 5.0, 6.0]) 

c = tf.add(a, b)
```

print(c)

>InvalidArgumentError: Multiple OpKernel registrations match NodeDef at the same priority '{{node AddV2}}': 'op: "AddV2" device_type: "GPU" constraint { name: "T" allowed_values { list { type: DT_FLOAT } } }' and 'op: "AddV2" device_type: "GPU" constraint { name: "T" allowed_values { list { type: DT_FLOAT } } }' [Op:AddV2]

Can I not use my GPU after all?

**Comment by PatriceVignola** [comment]
Hi @mariosconsta , did you mean `tensorflow-directml-plugin` instead of `tensorflow-directml`? If yes, this is already a known issue (https://github.com/microsoft/tensorflow-directml-plugin/issues/216). The solution at the moment is to use the `tensorflow-cpu` plugin together with `tensorflow-directml-plugin`, **NOT** `tensorflow` or `tensorflow-gpu`. We have a PR open against the TensorFlow Core repo that hopefully will get merge soon (https://github.com/tensorflow/tensorflow/pull/55645), so use `tensorflow-cpu` in the meantime.

**Functionally speaking, all packages are equivalent when using pluggable devices so your GPU will still be used, even though the package is named `tensorflow-cpu`.**
